kitsune/wiki/tests/test_templates.py

```diff
@@ -1620,6 +1620,30 @@ def test_self_approve_with_revision_contributors(self):
         assert rev1.creator.username in label
         assert u.username not in label
 
+    def test_review_passed_revision(self):
+        """Verify that its not possible to review a revision older than the current revision"""
+        r1 = revision(is_approved=False, save=True)
+        r2 = revision(document=r1.document, is_approved=True)
+        r2.save()
```

Comment:
I tried with that, but Travis is having this Error while I ran it locally
`kitsune/wiki/tests/test_templates.py:1626:9: F841 local variable 'r2' is assigned to but never used`
So I used `r2.save()` to use the r2 variable.